app/views/contexts/list_collector_context.py

```diff
@@ -0,0 +1,84 @@
+from flask import url_for
+from flask_babel import lazy_gettext
+
+from app.questionnaire.placeholder_renderer import PlaceholderRenderer
+from app.views.contexts.context import Context
+from app.views.contexts.question import build_question_context
+
+
+class ListCollectorContext(Context):
+    @staticmethod
+    def get_item_title(
+        list_block_summary, list_item_id, is_primary, placeholder_renderer
+    ):
+        rendered_summary = placeholder_renderer.render(list_block_summary, list_item_id)
+
+        if is_primary:
+            rendered_summary["item_title"] += lazy_gettext(" (You)")
+
+        return rendered_summary["item_title"]
+
+    def build_list_items_summary_context(self, list_collector_block, return_to=None):
+        list_name = list_collector_block["for_list"]
+        list_item_ids = self._list_store[list_name].items
+
+        primary_person = self._list_store[list_name].primary_person
+
+        placeholder_renderer = PlaceholderRenderer(
```

Comment:
You can just use self._placeholder_renderer here.

Reply:
Fixed this.